vhdbuilder/packer/list-images.sh

```diff
@@ -0,0 +1,16 @@
+#!/usr/bin/env bash
+set -euxo pipefail
+
+CONTAINER_RUNTIME="${CONTAINER_RUNTIME:-docker}"
+
+if [[ ${CONTAINER_RUNTIME} == "containerd" ]]; then
+    crictl images -o json
+    crictl images -o json | jq -c '[.images | .[] | {id:.id, repoTags:.repoTags, repoDigests:.repoDigests}]' > /opt/azure/containers/image-bom.json
+elif [[ ${CONTAINER_RUNTIME} == "docker" ]]; then
+    docker inspect $(docker images -aq) -f '{"id":"{{.ID}}","repoTags":{{json .RepoTags}},"repoDigests":{{json .RepoDigests}}}' | jq --slurp . > /opt/azure/containers/image-bom.json
```

Comment:
I was missing the dot here for jq, because I was on a version that didn't require it for stdin -_-